prow/plugins/label/label_test.go

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package label
+
+import (
+	"fmt"
+	"testing"
+
+	"k8s.io/kubernetes/pkg/util/sets"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/github/fakegithub"
+)
+
+const (
+	fakeRepoOrg  = "fakeOrg"
+	fakeRepoName = "fakeName"
+	orgMember    = "Alice"
+	nonOrgMember = "Bob"
+	prNumber     = 1
+)
+
+var areaLabels = []string{"area/api", "area/infra"}
+var priorityLabels = []string{"priority/important", "priority/critical"}
+
+func formatLabels(labels sets.String) sets.String {
```

Comment:
Yeah, didn't mean to offend you :-)
I do believe it will be much easier to read with `...string` though.